stable_baselines/a2c/utils.py

```diff
@@ -158,6 +158,11 @@ def linear(input_tensor, scope, n_hidden, *, init_scale=1.0, init_bias=0.0):
         return tf.matmul(input_tensor, weight) + bias
 
 
+def action_mask(input_tensor, mask_tensor, scope):
```

Comment:
I wouldn't call that function `action_mask` because this is a bit confusing, maybe `apply_mask`?

Reply:
+ missing docstring